[2025-01-03T07:35:10.969+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-03T07:35:10.985+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_processing.is_api_available scheduled__2025-01-02T00:00:00+00:00 [queued]>
[2025-01-03T07:35:10.995+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_processing.is_api_available scheduled__2025-01-02T00:00:00+00:00 [queued]>
[2025-01-03T07:35:10.996+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-03T07:35:11.018+0000] {taskinstance.py:2889} INFO - Executing <Task(HttpSensor): is_api_available> on 2025-01-02 00:00:00+00:00
[2025-01-03T07:35:11.028+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=75) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-03T07:35:11.031+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'scheduled__2025-01-02T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp3kdl1ph8']
[2025-01-03T07:35:11.032+0000] {standard_task_runner.py:72} INFO - Started process 93 to run task
[2025-01-03T07:35:11.033+0000] {standard_task_runner.py:105} INFO - Job 28: Subtask is_api_available
[2025-01-03T07:35:11.107+0000] {task_command.py:467} INFO - Running <TaskInstance: user_processing.is_api_available scheduled__2025-01-02T00:00:00+00:00 [running]> on host a67ecbd59b00
[2025-01-03T07:35:11.231+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='user_processing' AIRFLOW_CTX_TASK_ID='is_api_available' AIRFLOW_CTX_EXECUTION_DATE='2025-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-02T00:00:00+00:00'
[2025-01-03T07:35:11.234+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-03T07:35:11.235+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-03T07:35:11.236+0000] {logging_mixin.py:190} INFO - Current task name:is_api_available state:running start_date:2025-01-03 07:35:10.985817+00:00
[2025-01-03T07:35:11.236+0000] {logging_mixin.py:190} INFO - Dag name:user_processing and current dag run status:running
[2025-01-03T07:35:11.237+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-03T07:35:11.238+0000] {baseoperator.py:416} WARNING - HttpSensor.execute cannot be called outside TaskInstance!
[2025-01-03T07:35:11.238+0000] {http.py:137} INFO - Poking: api/
[2025-01-03T07:35:11.253+0000] {base.py:84} INFO - Retrieving connection 'user_api'
[2025-01-03T07:35:12.493+0000] {base.py:336} INFO - Success criteria met. Exiting.
[2025-01-03T07:35:12.508+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-03T07:35:12.509+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, run_id=scheduled__2025-01-02T00:00:00+00:00, execution_date=20250102T000000, start_date=20250103T073510, end_date=20250103T073512
[2025-01-03T07:35:12.533+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-03T07:35:12.534+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-03T07:35:12.535+0000] {logging_mixin.py:190} INFO - Dag name:user_processing queued_at:2025-01-03 07:35:05.385903+00:00
[2025-01-03T07:35:12.536+0000] {logging_mixin.py:190} INFO - Task hostname:a67ecbd59b00 operator:HttpSensor
[2025-01-03T07:35:12.586+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-03T07:35:12.611+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-03T07:35:12.614+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
